test/martha/martha_v3.test.js

```diff
@@ -857,24 +864,33 @@ test.serial('martha_v3 parses Kids First response correctly', async (t) => {
     t.is(response.statusCode, 200);
 
     const result = response.send.lastCall.args[0];
-    sinon.assert.callCount(getJsonFromApiStub, 2); // Bond was called to get SA key
-    t.deepEqual({ ...result }, kidsFirstDrsMarthaResult(googleSAKeyObject));
+    sinon.assert.callCount(getJsonFromApiStub, 3); // DRS metadata, Bond access token, DRS access URL
+    t.deepEqual({ ...result }, kidsFirstDrsMarthaResult(drsAccessUrlResponse));
     t.is(
         getJsonFromApiStub.getCall(0).args[0],
-        `https://${config.HOST_KIDS_FIRST_STAGING}/ga4gh/dos/v1/dataobjects/ed6be7ab-068e-46c8-824a-f39cfbb885cc`,
+        `https://${config.HOST_KIDS_FIRST_STAGING}/ga4gh/drs/v1/objects/ed6be7ab-068e-46c8-824a-f39cfbb885cc`,
     );
     t.falsy(getJsonFromApiStub.getCall(0).args[1]); // no auth passed
-    const requestedBondUrl = getJsonFromApiStub.getCall(1).args[0];
-    const matches = requestedBondUrl.match(bondRegEx);
-    t.truthy(matches, 'Bond URL called does not match Bond URL regular expression');
-    const expectedProvider = 'dcf-fence';
-    const actualProvider = matches[2];
-    t.is(actualProvider, expectedProvider);
+    const requestedBondAccessToken = getJsonFromApiStub.getCall(1).args[0];
```

Comment:
While making other edits, please add a newline between checks for `getCall(0)` and `getCall(1)` to match the spacing between `getCall(2)` below.